keyboards/pursuit40/keymaps/via/keymap.c

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2020 Nathan Spears
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+// Defines for task manager and such
+#define CALTDEL LCTL(LALT(KC_DEL))
+#define TSKMGR LCTL(LSFT(KC_ESC))
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* [0]
+     * ,-----------------------------------------------------------------------------------------.
+     * |   =        |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  |   -  |
+     * |------------+------+------+------+------+------+------+------+------+------+------+------|
+     * | Tab        |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Del  |
+     * |------------+------+------+------+------+-------------+------+------+------+------+------|
+     * | CTL & ESC  |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |  "   |
+     * |------------+------+------+------+------+------|------+------+------+------+------+------|
+     * | Shift      |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |Enter |
+     * |-------+-------+-------+-------+-------+-------+------+------+------+------+------+------|
+     * | PRINT   |  OS   |  Alt  |  Layer  |    Space & Layer     |   [     |   ]    |    CAPS   |
+     * `-----------------------------------------------------------------------------------------'
+     */
+    [0] = LAYOUT_default(
```

Comment:
This looks to have one more full line than the default, and the layout macro, and is erroring out due to this.

Reply:
Correct. Extra row deleted 👍